app/Http/Controllers/PostController.php

```diff
@@ -281,28 +299,13 @@ public function update(Request $request, $id)
         return redirect()->route('posts.show', $id);
     }
 
-
     public function destroy($id)
```

Comment:
can you use the same as approach as this?
 $user = $this->user->findOrFail($user_id);
        $user->delete();

Reply:
I change the approach to delete post.